src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlStatsAction.java

```diff
@@ -69,7 +69,7 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
             LOG.error("Failed during Query SQL STATS Action.", e);
 
             return channel -> channel.sendResponse(new BytesRestResponse(SERVICE_UNAVAILABLE,
-                    new ErrorMessage(e, SERVICE_UNAVAILABLE.getStatus()).toString()));
+                    new ErrorMessage<>(e, SERVICE_UNAVAILABLE.getStatus()).toString()));
```

Comment:
Using ErrorMessageFactory.createErrorMessage?

Reply:
It's done, thanks!